### PR TITLE
Add a banner to display a disclaimer

### PR DESCRIPTION
**Pull request by nonvis**

### CODE REVIEW

File: app/src/app/ProvidersAndLayout.tsx [hunk 1]
```diff
@@ -16,12 +17,20 @@ export const ProvidersAndLayout = ({ children }: ChildrenProps) => {
...
-          backgroundPosition: "bottom", 
+          backgroundSize: "cover",
+          backgroundPosition: "bottom",
           backgroundRepeat: "no-repeat",
         }}
       >
-        {children}
+        <div className="relative w-full h-screen flex-col">
+          <div className="sticky top-0 w-full flex justify-evenly items-center bg-white py-3 px-5">
+            <span className="text-opacity-90 text-[14px] text-[#4F4F4F]">[Mysten Solitaire] is provided for testnet purposes only and do not involve real money or the opportunity to win real money.</span>
```
Comment (Tzal3x) [inline app/src/app/ProvidersAndLayout.tsx:27]:
```suggestion
            <span className="text-opacity-90 text-[14px] text-[#4F4F4F]">[Mysten Solitaire] is provided for testnet purposes only and does not involve real money or the opportunity to win real money.</span>
```

**Review by Tzal3x — CHANGES_REQUESTED** [review]
Hello! Nice work 👍 
Two fixes: 
- typo on banner text
- white space on landing screen layout (see vercel deployment comment)

File: app/src/app/ProvidersAndLayout.tsx (hunk 1 above)
Comment (nonvis) [reply app/src/app/ProvidersAndLayout.tsx]: I've fixed the typo.

**Review by Tzal3x — APPROVED** [review]
Awesome, let's merge!

**Pull request merged**